CODE_GENERATION.md

```diff
@@ -92,7 +92,7 @@ of the provider. Create a file called `pkg/controller/<serviceid>/setup.go` and
 add the setup function like the following:
 ```golang
 // SetupStage adds a controller that reconciles Stage.
-func SetupStage(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter) error {
+func SetupStage(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter, poll time.Duration) error {
```

Comment:
We're getting pretty close to the point that I'd prefer to pass through a struct of setup dependencies rather than keep tacking on arguments to this function signature. Something like:

```go
func SetupStage(mgr ctrl.Manager, d setup.Dependencies) error { ... }
```

I could be convinced to punt on that until we find ourselves adding the 5th argument though.

Reply:
I think that would be a good idea, I almost wonder if we could encode some of this in `crossplane-runtime`. I want to think about this a bit so I prefer to defer it until later for now :+1: